### ISSUE TITLE
[UX2.0] basic parcel fails with Invalid Json Payload Input

### ISSUE THREAD

**Issue by tzarski0** [CiscoDevNet/terraform-provider-sdwan#223]
When configuring below resource:
```
resource "sdwan_system_basic_profile_parcel" "system_basic_profile_parcel" {
  config_description_variable = "{{system_description}}"
  feature_profile_id          = "76b21a53-b780-4e87-9652-71447378eb57"
  gps_latitude_variable       = "{{system_latitude}}"
  gps_longitude_variable      = "{{system_longitude}}"
  id                          = (known after apply)
  idle_timeout                = 300
  location_variable           = "{{system_location}}"
  name                        = "basic"
  on_demand_enable            = true
  on_demand_idle_timeout      = 3000
  timezone                    = "Europe/Warsaw"
  version                     = (known after apply)
}
```

it fails with:
```
│   with module.sdwan.sdwan_system_basic_profile_parcel.system_basic_profile_parcel["system1-basic"],
│   on ../terraform-sdwan-nac-sdwan/sdwan_profile_parcels.tf line 101, in resource "sdwan_system_basic_profile_parcel" "system_basic_profile_parcel":
│  101: resource "sdwan_system_basic_profile_parcel" "system_basic_profile_parcel" {
│ 
│ Failed to configure object (POST), got error: HTTP Request failed: StatusCode 400, {"error":{"message":"Invalid Json
│ Payload Input","code":"SCHVALID0001","details":"{\"Validation Errors\":{\"Required But Missing
│ Attributes\":[\"data.deviceGroups\",\"data.maxOmpSessions\"]}}","type":"error"}}
```

This was working few 1-2 weeks ago and now it's not.

**Comment by danischm** [comment]
https://github.com/CiscoDevNet/terraform-provider-sdwan/commit/189046b85dde395d3088bf191360b86235eefcb6

**Comment by tzarski0** [comment]
Fixed in v0.3.8 release.
